assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py

```diff
@@ -383,21 +495,17 @@ def __init__(self, **kwargs):
 
     def get_model_signature(self):
         """Return model signature for NLP models."""
-        if self._task == SupportedNLPTasks.QUESTION_ANSWERING.value:
-            return ModelSignature(
-                inputs=Schema(
-                    inputs=[
-                        ColSpec(name="question", type=DataType.string),
-                        ColSpec(name="context", type=DataType.string),
-                    ]
-                ),
-                outputs=Schema(inputs=[ColSpec(name="text", type=DataType.string)]),
-            )
-
-        return ModelSignature(
-            inputs=Schema(inputs=[ColSpec(name="input_string", type=DataType.string)]),
-            outputs=Schema(inputs=[ColSpec(name="text", type=DataType.string)]),
-        )
+        if self._task == SupportedNLPTasks.TEXT_GENERATION.value:
```

Comment:
does text2textgeneration also need to have similar params?

Reply:
Not that we know of at the moment